apps/files/src/components/FileListFilter/FileListFilterType.vue

```diff
@@ -71,6 +75,10 @@ export default defineComponent({
 	},
 
 	watch: {
+		/** Reset selected options if property is changed */
+		preset() {
```

Comment:
```suggestion
		presets() {
```
Looks like the prop was renamed to `presets`